src/main/java/teammates/ui/controller/InstructorCourseInstructorAddAction.java

```diff
@@ -197,7 +202,8 @@ private void removeSessionLevelPrivileges(InstructorAttributes instructorToAdd,
                     String valueForSectionName =
                             getRequestParamValue(Const.ParamsNames.INSTRUCTOR_SECTION_GROUP + i
                                                  + Const.ParamsNames.INSTRUCTOR_SECTION + j);
-                    if (isSectionGroupSpecial && valueForSectionName != null && sectionNamesTable.containsKey(valueForSectionName)) {
+                    if (isSectionGroupSpecial && valueForSectionName != null
+                            && sectionNamesTable.containsKey(valueForSectionName)) {
```

Comment:
The 4 spaces indentation seems to suggest that there are only two conditionals when there are in fact 3. Perhaps align the `&&` with `[i]sSectionGroupSpecial`